connect/api/src/main/java/org/apache/kafka/connect/sink/SinkTaskContext.java

```diff
@@ -79,4 +79,23 @@
      * @param partitions the partitions to resume
      */
     void resume(TopicPartition... partitions);
+
+    /**
+     * Whether to disable consumer offset commit in the framework. SinkTasks should use this if they manage offsets
```

Comment:
As mentioned elsewhere (but mentioning again here to make sure it makes it into the code), many of the method names should be linkified so you can click through to them in the generated javadocs.